lib/util/fs.js

```diff
@@ -137,7 +137,10 @@ const relative = (fs, rootPath, targetPath) => {
 		return fs.relative(rootPath, targetPath);
 	} else if (rootPath.startsWith("/")) {
 		return path.posix.relative(rootPath, targetPath);
-	} else if (rootPath.length > 1 && rootPath[1] === ":") {
+	} else if (
+		(rootPath.length > 1 && rootPath[1] === ":") ||
+		rootPath.startsWith("\\\\")
```

Comment:
```suggestion
		rootPath.length > 1 &&
		(rootPath[1] === ":" || rootPath.startsWith("\\\\"))
```

Reply:
Resolved by moving to using node functions